filteradapt/__init__.py

```diff
@@ -1,2 +1,4 @@
+__all__ = ["DataSet", "set_data_directory"]
```

Comment:
Can you comment on this line? My understanding of `__all__` is that it should neither be necessary nor change the current behaviour? Did Sphinx require it to correctly pick things up with autodoc?

Reply:
I can confirm that `autodoc` needs it, but I am quite unhappy with it. I added a bit of documentation about it.